app/src/main/java/ch/epfl/sweng/SDP/matchmaking/Matchmaker.java

```diff
@@ -64,6 +64,7 @@ private Matchmaker(Account account) {
         // TODO define a method in account that extracts directly the number corresponding
         // TODO to the league
         data.put("league", account.getCurrentLeague().replaceAll("\\D+", ""));
+        data.put("mode", 0);
```

Comment:
This should be data.put("mode", gameMode);

Reply:
Actually data.put("mode", gameMode); is written the next line, so this should be removed

Reply:
I think @Claudio5 wrote this

Reply:
True it did not merge correctly